openmetadata-ui/src/main/resources/ui/src/pages/explore/ExplorePage.component.tsx

```diff
@@ -188,55 +134,10 @@ const ExplorePage: FunctionComponent = () => {
     AppState.updateExplorePageTab(tab);
   }, [tab]);
 
-  useEffect(() => {
-    setSearchResult(undefined);
-
-    fetchData([
-      {
-        queryString: searchText,
-        from: INITIAL_FROM,
-        size: PAGE_SIZE,
-        filters: getFilterString(initialFilter),
-        sortField: initialSortField,
-        sortOrder: INITIAL_SORT_ORDER,
-        searchIndex: getCurrentIndex(tab),
-      },
-      {
-        queryString: searchText,
-        from: INITIAL_FROM,
-        size: ZERO_SIZE,
-        filters: getFilterString(initialFilter),
-        sortField: initialSortField,
-        sortOrder: INITIAL_SORT_ORDER,
-        searchIndex: getCurrentIndex(tab),
-      },
-      {
-        queryString: searchText,
-        from: INITIAL_FROM,
-        size: ZERO_SIZE,
-        filters: getFilterString(initialFilter),
-        sortField: initialSortField,
-        sortOrder: INITIAL_SORT_ORDER,
-        searchIndex: getCurrentIndex(tab),
-      },
```

Comment:
@aniketkatkar97, As we are removing this from here, are we handling the initial filters on the explore component?

Reply:
These calls were unnecessary as we are doing calls from component level.